spec/support/error_behavior.rb

```diff
@@ -1,7 +1,8 @@
 RSpec.shared_context 'an error' do
   it do
-    should raise_error(described_class) do |raised|
-      expect(raised.message).to eq(message)
+    expect { subject.call }.to raise_error do |raised|
+      expect(raised).to be_a(described_class)
```

Comment:
I would tend to use `be_instance_of` instead `be_a` or `be_kind_of`. It's a stronger check, and if it becomes untrue I would want to know.